model_compression_toolkit/core/common/graph/base_graph.py

```diff
@@ -91,6 +92,18 @@ def set_tpc(self,
         Args:
             tpc: TargetPlatformCapabilities object.
         """
+        # validate graph nodes are either from the framework or a custom layer defined in the TPC
```

Comment:
--> # Validate graph nodes are either built-in layers from the framework or custom layers defined in the TPC